cruise-control/src/test/resources/DefaultCapacityConfig.json

```diff
@@ -11,14 +11,14 @@
       "doc": "This is the default capacity. Capacity unit used for disk is in MB, cpu is in percentage, network throughput is in KB."
     },
     {
-      "brokerId": "314",
+      "brokerId": "1",
       "capacity": {
         "DISK": "150000",
         "CPU": "100",
         "NW_IN": "150000",
         "NW_OUT": "150000"
       },
-      "doc": "This overrides the capacity for broker 314."
```

Comment:
Nit: I am curious why this change is needed.

Reply:
This is because in many CC tests we use cluster model with broker id 0 and 1 in it.
 So all brokers are used to be estimated, now there will be one estimated and one accurate, the tests actually cover more scenarios automatically.